README.md

```diff
@@ -239,6 +247,10 @@ Crates tailored for general CPU architectures.
 - [`cortex-m`](https://github.com/japaric/cortex-m) Low level access to Cortex-M processors - ![crates.io](https://img.shields.io/crates/v/cortex-m.svg)
 - [`cortex-a`](https://github.com/andre-richter/cortex-a) Low level access to Cortex-A processors (early state) - ![crates.io](https://img.shields.io/crates/v/cortex-a.svg)
 
+### RISC-V
+
+- [`riscv`](https://github.com/rust-embedded/riscv) Low level access to RISCV processors - ![crates.io](https://img.shields.io/crates/v/riscv.svg)
```

Comment:
Could you change `RISCV` to `RISC-V` here? I forgot to fix this in `riscv` crate.

Reply:
I was wondering about that. Fixed.

Reply:
Thanks!